src/library/process.js

```diff
@@ -70,10 +87,16 @@ export const processMessage = async (AWS, region, service, account,
   }, msgHandler);
   console.log(`processMessage: INFO: Result from worker is ${JSON.stringify(procRes, null, 4)}`);
 
+  // CR: Mickey: is the assumption below that the serviceUserData + account found in the previous
+  //    block (ln43-62) may now be out of date?  
+  //    I would also lean towards making this a named helper function to make it more
+  //    clear what its purpose is like "saveServiceDataToAccount"
```

Comment:
@MickeyMcc This is to check if the worker wants to modify the serviceUserData after the current execution and if yes, modify it in the database. This will only modify the service specific data in the account and will not touch anything else in the account.

Reply:
@MickeyMcc Just fixed the issue with getting serviceUserData from the worker result.